src/actions/bitcoinActions.js

```diff
@@ -430,8 +431,8 @@ export const addBTCAssetsAction = () => {
       assets: { data: assets, supportedAssets },
       bitcoin: { data: { addresses } },
     } = getState();
-    if (supportedAssets && !supportedAssets.some(e => e.symbol === 'BTC')) {
-      const btcAsset = initialAssets.find(e => e.symbol === 'BTC');
+    if (supportedAssets && !supportedAssets.some(e => e.symbol === BTC)) {
+      const btcAsset = initialAssets.find(e => e.symbol === BTC);
```

Comment:
I can see `e => e.symbol === BTC` repeating a lot in the code, what do you think about adding basic filter method `isBTCAsset` that would accept `Asset` and check its symbol?